### ISSUE TITLE
Fresh git clones show modified CSV and/or JS files

### ISSUE THREAD

**Issue by toolness** [18F/calc#1414]
This is really weird and I can't tell if it's something strange on my system, or something strange with our repo.

Basically, when I do a fresh checkout of our repo, via `git clone https://github.com/18F/calc`, `cd` into it and run `git status`, I get this:

> <img width="462" alt="screen shot 2017-02-21 at 5 20 31 pm" src="https://cloud.githubusercontent.com/assets/124687/23187775/19b995f2-f85a-11e6-9770-8ea05ed8656e.png">

Wut.

This might have something bizarre to do with our `.gitattributes` but I'm not sure; doing a `git diff` on the files shows all the lines removed but then added with no apparent differences, which could mean that the line endings just changed.

@jseppi can you reproduce this on your end? If not, it's probably something wonky with my system.

**Comment by jseppi** [comment]
Yeah it actually just happened to me today also. It definitely has to do with `.gitattributes`. I made a local change to mine to mark csv as binary just as a temporary fix. I was so stuck I couldn't switch branches at all. Not sure what the corrective action is yet -- I've been unable to reproduce. I've been having a lot of trouble actually dealing with our CSVs this morning, both due to this, and due to weird unicode issues that seem to be coming from Excel.

Maybe need to add `*.csv eol=crlf`?

**Comment by toolness** [comment]
Also, if I run `git stash`, I get a bunch of messages like the following:

```
warning: CRLF will be replaced by LF in contracts/docs/hourly_prices_sample.csv.
The file will have its original line endings in your working directory.
```

This seems to suggest that the files in the repository might somehow have CRLF endings, even though `.gitattributes` expects them all to have LF endings, and git could just be massively confused about it.

Also, `git stash` doesn't actually result in an unmodified git state; running `git status` again shows the same files as being modified.

**Comment by toolness** [comment]
Ah ok!  Yeah, if it's Excel that's making those CSVs, I guess it makes sense to either add `*.csv eol=crlf` *or* alternatively convert the line endings to LF's before committing them.  I think the latter could be done by following the [Refreshing a repository after changing line endings](https://help.github.com/articles/dealing-with-line-endings/#refreshing-a-repository-after-changing-line-endings) instructions.

**Comment by jseppi** [comment]
Yup, that `git stash` problem was happening to me as well.

**Comment by toolness** [comment]
Hmm this seems like it might be a good thing to screenhero on, since it is baffling both of us.

**Comment by jseppi** [comment]
Agree, but I am heading to airport now :(

**Comment by toolness** [comment]
Fixed by #1465!  The `.csv` files are dealt with, hopefully those `.js` files won't come back to haunt us.
